content/en/tracing/trace_collection/automatic_instrumentation/compile_time/go.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Automatic Compile-Time Instrumentation for Go
+kind: documentation
+is_beta: true
+code_lang: go
+# type: multi-code-lang
+# code_lang_weight: 20
+further_reading:
+- link: https://datadoghq.dev/orchestrion
+  tag: Documentation
+  text: Reference documentation
+- link: https://github.com/DataDog/orchestrion/discussions
+  tag: GitHub
+  text: Discuss on GitHub
+- link: https://github.com/DataDog/orchestrion/releases
+  tag: GitHub
+  text: Releases
+- link: https://github.com/DataDog/orchestrion
+  tag: GitHub
+  text: Source code
+---
+
+<div class="alert alert-info">
+
+  This feature is under active development.
+
+  Early adopters are encouraged to provide feedback via any of the following channels:
+  - [GitHub issues][1]
+  - [This form][2]
+  - [GitHub discussions][3]
+
+</div>
+
+## Introduction
+
+Orchestrion is Datadog's automatic compile-time instrumentation tool for Go applications. It automatically adds instrumentation (backed by the [Go Datadog Trace library][4]) to Go applications during compilation.
+
+Running at compile-time allows Orchestrion to not only instrument your own code, but also all of your dependencies' as
+well as the Go standard library, ensuring you get the best coverage possible.
+
+## Compatibility
+
+### Go language
+
+Orchestrion supports the two latest releases of Go, matching Go's [official release policy][5]. Older releases may function, but are not supported by default. For special requests, [contact support][6].
+
+Orchestrion only supports [go modules][7].
+
+### Framework Support
+
+While the goal is to eventually support all integrations available in the [Go Datadog Trace library][4], support is currently only built for a gowing subset of them. The complete list of supported frameworks can be found in the [documentation][8].
+
+## Getting Started
+
+### Installation
+
+1. Install Orchestrion in your environment:
+   ```sh
+   go install github.com/datadog/orchestrion@latest
+   ```
+2. If necessary, also add `GOBIN` to your `PATH`:
+   ```sh
+   export PATH="${PATH}:$(go env GOBIN)"
+   ```
+3. Register Orchestrion in your project's `go.mod` to ensure reproductibe builds:
+   ```sh
+   orchestrion pin
+   ```
+4. Commit changes to your version control system:
+   ```sh
+   git commit -m "chore: enable orchestrion" go.mod go.sum orchestrion.tool.go
+   ```
+
+### Usage
+
+Adjust your build commands or environment to be Orchestrion-enabled using either of these options:
+
+- **Use `orchestrion go`:** Simply prepend all your usual `go` commands with `orchestrion`. For example:
+  ```sh
+  orchestrion go build .
+  orchestrion go run .
+  orchestrion go test ./...
+  ```
+- **Add a `-toolexec` argument:** Manually add the required `-toolexec="orchestrion toolexec"` argument to your `go` commands:
```

Comment:
I think toolexec requires the fullpath, no?

Reply:
No it can get from `$PATH`